hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/service/model/ContractCallDebugServiceParameters.java

```diff
@@ -0,0 +1,25 @@
+package com.hedera.mirror.web3.service.model;
+
+import com.hedera.mirror.web3.viewmodel.BlockType;
+import com.hedera.node.app.service.evm.store.models.HederaEvmAccount;
+import lombok.Builder;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.datatypes.Address;
+
+@Value
+@Builder
```

Comment:
It's used by the `OpcodeController` tests
```suggestion
@Builder(toBuilder = true)
```